src/client.ts

```diff
@@ -545,4 +608,21 @@ export default class ConcordiumNodeClient {
             });
         });
     }
+
+    createInstanceInfo(
```

Comment:
This method should not be exposed. It should just be a private method that is not shared in the client, or you can just move the actual implementation directly inside `getInstanceInfo`.

Reply:
You still have to remove this method.